pkg/controller/storagecluster/uninstall_reconciler_test.go

```diff
@@ -212,3 +212,47 @@ func assertGetStorageClusterEligibleNodes(
 	assert.NotEqual(t, 0, len(nodes.Items))
 	return nodes
 }
+
+func TestDeleteNodeTaint(t *testing.T) {
```

Comment:
We're moving everything to Table Driven approach. Would save us some re-factoring effort if you could follow it from the beginning.

Reply:
Can you point me to one particular TC which is using it.

Reply:
@iamniting - e.g. the test case I reviewed recently: `TestSetRookUninstallandCleanupPolicy`. You put the cases into a table (array of structs) and had the code just iterate over the array instead of explicitly writing out each case.

Reply:
Done